templates/cfp/installation_support.html

```diff
@@ -0,0 +1,100 @@
+<br>
+
+<h2 id="inspiration">Inspiration</h2>
+<p>If you're stuck and need inspiration, the EMF theme is <strong>'2022 Leagues Under The Sea'</strong>. Think Jules Verne, think steampunk, ...but cyber.</p>
+<p>If that doesn’t do it for you, we have <a href="https://wiki.emfcamp.org/wiki/Installation_Inspiration">a list of attendee-suggested projects</a> that might spark an idea - they're meant to be interpreted loosely!</p>
+<p>If you have half an idea, <a href="#contact">contact us</a>. Maybe we can match you up with someone.</p>
+<br>
+
+<h2 id="funding">Funding</h2>
+<p>We have a limited amount of funding available for installations - if you can't afford to make your idea happen, we may be able to help!</p>
+<p>We’re expecting grants to be in the range £100-£500, but larger grants may be possible for exceptional projects.</p>
+<p>If you'd like to apply for funding, please submit your installation to the <a href="{{ url_for('cfp.form', cfp_type='installation') + ('?closed' if ignore_closed else '') }}">Call For Participation</a>.</p>
+<br>
+
+
+<h2 id="funding">Tickets</h2>
+<p>Bringing an installation does not automatically get you a guaranteed EMF ticket, unless we’ve funded it. We have a limited number of guaranteed tickets for exceptional installations available.</p>
+<p>If you'd like to apply for a guaranteed EMF ticket, please submit your installation to the <a href="{{ url_for('cfp.form', cfp_type='installation') + ('?closed' if ignore_closed else '') }}">Call For Participation</a>.</p>
+<br>
+
+
+<h2>FAQ</h2>
+
+<h4>I’ve already got a thing, can I just bring it?</h4>
+<p>Yes! You don’t need our agreement, just bring it along. But <a href="#contact">we’d really like to know about it</a>, if only so we can ensure people can find it on-site. And if there is any question at all about safety or logistics, please get in touch.</p>
+
+<h4>How do I apply?</h4>
+<p>Fill in the <a href="{{ url_for('cfp.form', cfp_type='installation') + ('?closed' if ignore_closed else '') }}">Call for Participation</a>, and you have any questions about the application process, please <a href="#contact">get in touch</a>.</p>
+
+<h4>When can I expect a decision?</h4>
+<p>We’ll be reviewing submissions every few weeks. Funded projects will be awarded on April 4th. If you need money further in advance, please <a href="#contact">contact us</a>.
+
+<h4>What if my thing needs to be indoors?</h4>
+<p>We can host a limited number of installations in the Lounge, the Arcade, and Null Sector. <a href="#contact">Get in touch</a> and we’ll see if we can make it work.</p>
+
+<h4>What if it doesn’t work / isn’t lovely / I don’t finish it?</h4>
+<p>We'd like to see enough evidence that the idea is feasible and you can built it before we commit to funding. But if things go wrong, or if you run out of time to build it - don't worry. Just make sure you let us know so we can re-allocate the funding to someone who can use it.</p>
```

Comment:
Is it worth trimming the final sentence to `Just make sure you let us know` as if they've already spent the money we allocated ...